src/Symfony/Component/Messenger/Bridge/MongoDB/Tests/Stubs/mongodb.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace MongoDB {
+    if (!class_exists(Collection::class)) {
+        class Collection
+        {
+            public function deleteOne($filter, array $options = [])
+            {
+            }
+
+            public function findOneAndUpdate($filter, $update, array $options = [])
+            {
+            }
+
+            public function insertOne($document, array $options = [])
+            {
+            }
+        }
+    }
+
+    if (!class_exists(Database::class)) {
+        class Database
+        {
+            public function selectCollection(string $collectionName, array $options = [])
```

Comment:
This method was renamed in `mongob/mongodb: 1.21`.
```suggestion
            public function getCollection(string $collectionName, array $options = [])
```

Reply:
Oh, that could be a reason to bump the required extension... I'll leave to the core team this decision and adapt accordingly.